src/validations/modelPlanCollaborators.ts

```diff
@@ -6,6 +6,7 @@ const CollaboratorsValidationSchema = Yup.object().shape({
   }),
   teamRoles: Yup.array()
     .of(Yup.string())
+    .min(1, 'Requires at least one role')
     .required('Enter a role for this team member')
 });
```

Comment:
This code ensures that there is at least one role in the `teamRoles` array. I think this is the simplest FE method of addressing the issue you brought up, @patrickseguraoddball . Thoughts?

Reply:
Yep this is great thanks!  I think BE is also working on a restraint for this, but it's best to have it in both places